Kernel/Arch/aarch64/RPi/GPIO.cpp

```diff
@@ -73,7 +72,7 @@ void GPIO::internal_enable_pins(u32 enable[2], PullUpDownState state)
     m_registers->pull_up_down_enable = static_cast<u32>(state);
 
     // 2. Wait 150 cycles – this provides the required set-up time for the control signal
-    wait_cycles(150);
+    Kernel::Aarch64::Asm::wait_cycles(150);
```

Comment:
Please make sure to drop the `Kernel` prefix when renaming the namespace, same below

Reply:
Fixed!